### PR TITLE
docs: fix install instructions docker

### PR DESCRIPTION
**Pull request by JoanFM**
**Goals**
Step-by-step following of installation instructions do not work on Docker

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -25,18 +25,28 @@ docker pull jinaai/jina:latest
...
+```shell
+docker run -it --entrypoint=/bin/bash jinaai/jina:latest -p 54321:54321
+jina new hello-jina
+cd hello-jina
+jina flow --uses flow.yml
+```
+````
 
-And observe the result from your terminal.
+Run the client in your machine and observe the result from your terminal.
```
Comment (alexcg1) [inline docs/index.md:46]:
```suggestion
Run the client on your machine and observe the results from your terminal.
```

File: docs/index.md [hunk 2]
```diff
@@ -25,18 +25,28 @@ docker pull jinaai/jina:latest
 ```
 ````
 
-Now that you’re set up, let’s create a project:
+Now that you’re set up, let’s create a project, and start a project to serve:
```
Comment (alexcg1) [inline docs/index.md:28]: I think this part was fine before. Doesn't need a change

**Review by alexcg1 — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 2 above)
Comment (JoanFM) [reply docs/index.md:28]: then propose to remove and I can add it in the batch

Comment (JoanFM) [reply docs/index.md]: Done

**Review by alexcg1 — APPROVED** [review]
LGTM :+1:

**Pull request merged**